### PR TITLE
Update Ember guidelines

### PR DESCRIPTION
**Pull request by hakubo**
- remove guidelines for server
- add info about destructuring

@Wikia/x-wing @Wikia/west-wing @Wikia/spitfires @kvas-damian @bkoval @rafalkalinski

### CODE REVIEW

File: Ember/README.md [hunk 1]
```diff
@@ -27,115 +27,67 @@ app/components/my-components.js
...
 
 ```javascript
 // bad
-var FooView = Ember.View.extend();
+const FooView = Ember.View.extend();
 
 // good
-var FooView = Em.View.extend();
+import Ember from 'ember';
+const {View} = Ember;
```
Comment (rogatty) [inline Ember/README.md]: Can we not use `View` as a good example? ;)

**Comment by rogatty** [comment]
We already had a discussion with @dianafa if the server guidelines should live in the Ember file and came to conclusion that there is currently no better place for them. These rules are still valid and probably should be _somewhere_ until we move to Fastboot. If you insist on removing them from Ember ones, then I suggest creating some other document for them.

File: Ember/README.md [hunk 2]
```diff
@@ -144,27 +96,30 @@ In Ember objects, declare in this order:
 * Calculated property declarations
 * `actions` hash
 * `observer` declarations
-* Overloaded functions (`click`/`mouseUp` handlers in `Em.View` objects, etc.)
+* Overloaded functions (`click`/`mouseUp` handlers in `Ember.View` objects, etc.)
```
Comment (rogatty) [inline Ember/README.md]: This is the last `Ember.View` I found :wink:

**Comment by dianafa** [comment]
are we planning to move somewhere the guidelines for server or create a separate place for this?

**Comment by TK-999** [comment]
Closing as an old PR.

**Pull request closed without merging**